scripts/lib/CIME/provenance.py

```diff
@@ -23,6 +23,8 @@ def _get_batch_job_id_for_syslog(case):
             return os.environ["SLURM_JOB_ID"]
         elif mach in ['mira', 'theta']:
             return os.environ["COBALT_JOBID"]
+        elif mach in ['summit']:
```

Comment:
I understand that this is just extending bad code already present, but we shouldn't need to check the machine name here, instead we should get case.get_value(BATCH_SYSTEM)

Reply:
@worleyph